demo/app/lib/wallet_sdk/wallet_sdk_mobile.dart

```diff
@@ -65,6 +66,12 @@ class WalletSDK extends WalletPlatform {
     }
   }
 
+  Future<WalletSDKError> parseWalletSDKError({required String localizedErrorMessage}) async {
+    var parsedWalletError = await methodChannel.invokeMethod(
+        'parseWalletSDKError', <String, dynamic>{'localizedErrorMessage': localizedErrorMessage});
```

Comment:
Can this map be made more specific?

Reply:
You mean map of string , string ? Instead of dynamic 

Other than that walletsdkerror is the object type we are rendering the object which we get from sdk. 

I believe, It cant be more specific than that. I will change that if thats what u mean

Reply:
Even better: see if it can just be a simple string instead of a map